app/graphql/connections/base_connection.rb

```diff
@@ -3,7 +3,7 @@ class BaseConnection < GraphQL::Types::Relay::BaseConnection
     field :total_count, Integer, null: false
 
     def total_count
-      object.nodes.size
+      object.items&.size
```

Comment:
Shouldn't be this converted to `0` instead of `nil`?

Reply:
I'm not sure if it is possible that this is `nil`, but yes, you are right